### PR TITLE
Enable copr builds and add packit config

### PR DESCRIPTION
**Pull request by dhodovsk**
Let us introduce [packit service](https://packit.dev) to you - the automation to integrate upstream open source projects into Fedora operating system.

After merging this PR, you are just a few steps away from RPM builds being automatically triggered on your PR's.
It means, that you'll be able to try and play with your change, packaged as an RPM.

But there is more. By using packit, you can for example enable adding new releases into Fedora Rawhide.

What are the next steps?
* Install [Packit-as-a-Service github application](https://github.com/marketplace/packit-as-a-service) in your repo
* In case you are first user, wait for account approval
* Enjoy the built RPMs!

For more info, please:
 * check out the documentation: https://packit.dev/docs/
 * contact [@packit-service/the-packit-team](https://github.com/orgs/packit-service/teams/the-packit-team)

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+downstream_package_name: anaconda-user-help
```
Comment (vojtechtrefny) [inline .packit.yaml]: This should be `downstream_package_name: blivet-gui`, right? And same with the `synced_files` below.

**Comment by vojtechtrefny** [comment]
Jenkins, ok to test.

**Comment by vojtechtrefny** [comment]
I was already looking at packing before and I like the idea and I was definitely planning to use packit for Fedora builds. But I'm not sure if it's necessary to do a copr build for every PR. Our test suite already does mock builds for every PR to make sure building RPMs works and using Copr looks like an overkill for this. And I also don't want to add more jobs to already overloaded Copr infrastructure.

File: .packit.yaml (hunk 1 above)
Comment (dhodovsk) [reply .packit.yaml]: Oh, there was a mixup. I just mindlessly got the source from here https://src.fedoraproject.org/rpms/anaconda-user-help/blob/master/f/anaconda-user-help.spec :) It should be fixed now

**Comment by dhodovsk** [comment]
We plan to support other triggers for copr build - see https://github.com/packit-service/packit-service/issues/85.

There are other packit features that might be beneficial right now. For the list see 
 [packit cli commands](https://packit.dev/docs/cli/) or [Packit-as-a-service-doc](https://packit.dev/packit-as-a-service/)

**Comment by vojtechtrefny** [comment]
Thank you and thank you for the links, I'll definitely take a look at the other packit options.

**Pull request merged**